components/ProgressView/tests/unit/ProgressViewTests.m

```diff
@@ -41,13 +41,13 @@ - (void)testInitialProgress {
 }
 
 - (void)testSetProgress {
-  _progressView.progress = 0.1234f;
-  XCTAssertEqual(_progressView.progress, 0.1234f);
+  _progressView.progress = (float)0.1234;
```

Comment:
are these changes needed?

Reply:
Yes, it was previously using the `0.1f` syntax.